CODEOWNERS

```
@@ -0,0 +1,57 @@
+# Borrowed from https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+#
+# Note: @philCryoport and @theschles are the same human being.  @philCryoport is his work Github account; @theschles is his personal Github account.
+* @Pakisan @derberg @theschles @philCryoport
```

```suggestion
* @Pakisan @asyncapi-bot-eve @theschles @philCryoport
```

@Pakisan I guess I'm not needed anymore? right? 

we need bot in CODEOWNERS to cover automation

---

> we need bot in CODEOWNERS to cover automation
Agree

> I guess I'm not needed anymore? right?
I'm used to you approving my requests, will miss you 😄

---

@derberg if you can be there as a fallback should I be unavailable, I would ask that you stay on as well.

I'm going to be on vacation for approximately 2-1/2 weeks this summer -- and I'm trying to _not_ bring a computer with me.  

There's also always a chance that I'll be swamped and unable to review changes.